ClipboardAPI/clipboard-change-event-explainer.md

```diff
@@ -0,0 +1,197 @@
+
+#### 5.2.2 Approach 2 - No focus requirement
+
+##### Pros:
+1. Opens possibility for more user scenarios - e.g. For example, a web app can make network calls in the background when a specific clipboard item is updated - before pasting an image, a web app might want to check the image for embedded malicious content using a remote service. If the app can be notified about a clipboard change in background, it can trigger the necessary network operations while the user is in another page or application. When the user returns, the web app is ready with the results, reducing the wait time.
+
+##### Cons:
+1. Might be open to misuse - a web app will be able to monitor the clipboard even when the user is interacting with other applications / pages.
+2. May not be useful unless the page focus requirement is also removed from the async read/write clipboard API.
+3. Could lead to higher resource consumption due to continuous monitoring. E.g. if a large number of pages are listening to the clipboardchange event, when the clipboard is changed, then all the pages will receive the event at once, which might put load on the system.
+
+#### 5.2.3 Approach 3 - Transient user activation
+This approach allows the clipboardchange event to be fired for a short duration after the user loses page focus, such as up to 5 seconds. This ensures that clipboard changes occurring immediately after focus loss are still captured, enhancing user experience without compromising security.
+
+##### Pros:
+1. Clipboard changes occurring immediately after the user loses focus are still captured, ensuring the web app can respond promptly when the user returns.
+Example: A web app can pre-process clipboard data while the user is in another application, reducing wait time when the user comes back.
+2. Limits the duration for which clipboard monitoring is allowed after focus loss, reducing the risk of prolonged unauthorized access.
+
+##### Cons:
+1. The short duration might not be sufficient for some use cases where clipboard changes occur after the specified time.
+2. Still requires monitoring for a brief period after focus loss, which could lead to resource usage if many pages implement this.
+Example: Multiple tabs monitoring clipboard changes for 5 seconds could still cause a temporary spike in resource usage.
+
+#### 5.2.4 Conclusion
+We favour Approach 1 - Page required to be in focus to receive event, since this approach is inline with the current Async clipboard APIs and also reduces the possibility of misusing the clipboard change event in privacy related attacks. Also the approach has relatively lower resource usage.
+
+### 5.3 Event details 
+Since the clipboardchange event is not triggered by a user action and the event is not associated to any DOM element, hence this event doesn't bubbles and is not cancellable.
```

Comment:
Nit: did you mean `doesn't bubbles` -> `doesn't bubble up`?

Reply:
Yes, thanks for pointing